packages/node_modules/@webex/plugin-meetings/src/reachability/index.js

```diff
@@ -0,0 +1,283 @@
+
+/*!
+ * Copyright (c) 2015-2018 Cisco Systems, Inc. See LICENSE file.
+ */
+
+/* globals window */
+import {StatelessSparkPlugin} from '@ciscospark/spark-core';
+
+import peerConnectionManager from '../peer-connection-manager';
+import {ICESTATE, MEETINGS, REACHABILITY, _ANSWER_} from '../constants';
+
+import ReachabilityRequest from './request';
+
+/**
+ * @class Reachability
+ */
+export default class Reachability extends StatelessSparkPlugin {
+  namespace = MEETINGS;
+
+  constructor(attrs, options) {
+    super({}, options);
+    this.reachabilityRequest = new ReachabilityRequest({}, options);
+    this.gatherReachability();
+  }
+
+  setup = () => {
+    this.processedClusterResults = [];
+    this.peerConnectionMap = {};
+  }
+
+  /**
+   * fetches reachability data
+   * @returns {object} reachability data
+   */
+  gatherReachability = () => {
+    this.setup();
+    // Reset reachability result to empty to ensure no stale data
+    try {
+      window.localStorage.setItem(REACHABILITY.localStorage, JSON.stringify({}));
```

Comment:
@arun3528 - we want to use the storage adapter here most likely instead of going straight to local storage.

Reply:
@andrewholsted do you mean creating a storage adapter of using one from the SDK . I couldn't  any plugin for samples . The storage adapter was in web client do we want the same kind of file ?